src/clj/muuntaja/format/json.clj

```diff
@@ -35,23 +35,23 @@
           options)
         (.flush output-stream)))))
 
-;; muuntaja.json - experimental
+;; jsonista - experimental
```

Comment:
This could be in a separate-file? `json.clj` and `jsonista-json.clj`? or maybe `cheshire.clj` and `jsonista.clj`?